nat/src/main/java/org/hyperledger/besu/nat/kubernetes/KubernetesNatManager.java

```diff
@@ -48,8 +45,8 @@
 public class KubernetesNatManager extends AbstractNatManager {
   protected static final Logger LOG = LogManager.getLogger();
 
-  private static final String KUBE_CONFIG_PATH_ENV = "KUBE_CONFIG_PATH";
-  private static final String DEFAULT_KUBE_CONFIG_PATH = "~/.kube/config";
+  private static final String DEFAULT_SERVICE_NAMESPACE = "pegasys";
```

Comment:
I suggest no to use pegasys as default namespace value. Rather use besu. And maybe make it configurable through env variable